### PR TITLE
Feature/chioma/signin signup forms

### PR DESCRIPTION
**Pull request by Simpcyclassy**

### CODE REVIEW

**Comment by Way2nnadi** [comment]
You should only have one `constants.js` per component. Merge `signup/components/constants.js` into `signup/constants.js`. This will then require you to update the paths for files that are requiring your `constants.js` files.

**Comment by Simpcyclassy** [comment]
> You should only have one `constants.js` per component. Merge `signup/components/constants.js` into `signup/constants.js`. This will then require you to update the paths for files that are requiring your `constants.js` files.

This has been fixed

File: src/components/signup/components/SignupForm.js [hunk 1]
```diff
@@ -47,8 +76,47 @@ class SignupModal extends React.Component {
...
+                                            (input, option) => 
+                                                option.props.children.toLowerCase().indexOf(input.toLowerCase()) >= 0
+                                        }
+                                >
+                                    {createCategories}
+                                </Select>
+                            )
+                        }
+                    </FormItem>
+                    }
```
Comment (enyeInc) [inline src/components/signup/components/SignupForm.js]: You deleted all of the worked that Eric did to hook this component up to redux

File: src/components/signup/components/style.css [hunk 2]
```diff
@@ -4,8 +4,24 @@
...
   .agreementQuote {
-    font-size: 15px;
-    margin-left: 24px;
+    font-size: 14px;
+    margin-left: 20px;
     margin-top: 5px;
     margin-bottom: 5px;
   }
+
+  .radiogroupform{
```
Comment (enyeInc) [inline src/components/signup/components/style.css]: minor spaces error

File: src/components/signup/components/style.css [hunk 3]
```diff
@@ -4,8 +4,24 @@
...
+    font-size: 20px;
+    font-weight: bold;
+    color: #006400;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+  }
+
+  .radiobuttons{
```
Comment (enyeInc) [inline src/components/signup/components/style.css]: minor spacing error

File: src/components/signup/constants.js [hunk 4]
```diff
@@ -85,4 +93,58 @@ export const INPUTS = [
...
+    },
+    {
+        category: "Wheat",
+        value: "wheat",
+    },
+    {
+        category: "Other",
+        value: "other",
+    },
+];
```
Comment (enyeInc) [inline src/components/signup/constants.js]: good use of object of arrays here. You are starting to get it

File: src/components/signup/components/SignupForm.js [hunk 5]
```diff
@@ -47,8 +76,47 @@ class SignupModal extends React.Component {
...
+                    </FormItem>
+                    {isProducer &&
+                    <FormItem
+                        {...formItemLayout}
+                        label={PRODUCT_TYPE}
+                    >
+                        {
+                            getFieldDecorator('productType', {
+                                rules:
+                                [{ message: { MESSAGE }, required: true }],
```
Comment (enyeInc) [inline src/components/signup/components/SignupForm.js]: There is something wrong with this line that is causing your code to have errors. find it and fix it.

File: src/components/signin/components/Signin.js [hunk 6]
```diff
@@ -22,7 +22,7 @@ export default class Signin extends React.Component {
         const form = this.formRef.props.form;
         form.validateFields(err => {
             if (err) {
-                return;
+                return err;
             }
```
Comment (enyeInc) [inline src/components/signin/components/Signin.js]: I would rather you spelled out `err` as `error`

File: src/components/signup/components/SignupForm.js [hunk 7]
```diff
@@ -1,13 +1,25 @@
...
 
-import { formItemLayout, INPUTS, SIGNUP_STRINGS } from '../constants';
+import { formItemLayout, INPUTS, PRODUCTS, SIGNUP_STRINGS } from '../constants';
 
 const FormItem = Form.Item;
+const Option = Select.Option;
 const {
+    CATEGORIES,
+    CLASSNAME_ITTEM1,
+    CLASSNAME_RADIOBUTTONS,
```
Comment (enyeInc) [inline src/components/signup/components/SignupForm.js]: `CLASSNAME_RADIO_BUTTONS`

File: src/components/signup/components/SignupForm.js [hunk 8]
```diff
@@ -1,13 +1,25 @@
...
+import { Form, Input, Modal, Radio, Select } from 'antd';
 
-import { formItemLayout, INPUTS, SIGNUP_STRINGS } from '../constants';
+import { formItemLayout, INPUTS, PRODUCTS, SIGNUP_STRINGS } from '../constants';
 
 const FormItem = Form.Item;
+const Option = Select.Option;
 const {
+    CATEGORIES,
+    CLASSNAME_ITTEM1,
```
Comment (enyeInc) [inline src/components/signup/components/SignupForm.js]: `CLASSNAME_ITTEM_1`

File: src/components/signup/components/style.css (hunk 2 above)
Comment (Simpcyclassy) [reply src/components/signup/components/style.css]: Done

File: src/components/signup/constants.js (hunk 4 above)
Comment (Simpcyclassy) [reply src/components/signup/constants.js]: Thanks

File: src/components/signin/components/Signin.js (hunk 6 above)
Comment (Simpcyclassy) [reply src/components/signin/components/Signin.js]: Done

**Pull request merged**